client/ayon_core/hosts/harmony/plugins/publish/collect_farm_render.py

```diff
@@ -177,7 +177,8 @@ def get_instances(self, context):
                 outputFormat=info[1],
                 outputStartFrame=info[3],
                 leadingZeros=info[2],
-                ignoreFrameHandleCheck=True
+                ignoreFrameHandleCheck=True,
+                # deadline=inst.data.get("deadline")  TODO
```

Comment:
Any reason this is still `todo`? Due to old publisher UI? What's needed to implement this?

Reply:
Updated todo, will be reworked when new Publisher gets implemented.